docs/workshops-materials.md

```diff
@@ -0,0 +1,54 @@
+---
+title: Workshops' materials
+date: 2024-03-21
+sidebar_position: 13
+---
+
+# Workshops' materials
```

Comment:
```suggestion
# Workshop materials
```

Reply:
We could also use `Materials from Packit workshops`, but I think simple `Workshop materials` is the best.